test/rules/quotemarkRuleTests.ts

```diff
@@ -35,4 +36,16 @@ describe("<quotemark>", () => {
         assert.equal(actualFailures.length, 1);
         assert.isTrue(actualFailures[0].equals(expectedFailure));
     });
+
+    it("enforces single quotes but allow other quote marks to avoid having to escape", () => {
+        const actualFailures = Lint.Test.applyRuleOnFile(fileNameForEscape, QuoteMarkRule, [true, "single", "escape"]);
+
+        assert.equal(actualFailures.length, 0);
+    });
+
+    it("enforces double quotes but allow other quote marks to avoid having to escape", () => {
+        const actualFailures = Lint.Test.applyRuleOnFile(fileNameForEscape, QuoteMarkRule, [true, "double", "escape"]);
+
+        assert.equal(actualFailures.length, 0);
+    });
```

Comment:
these changes will need to be refactored when you move the test cases to `quotemark.test.ts`